src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Utilities/FilePathUtilties.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Utilities
+{
+    internal static class FilePathUtilities
```

Comment:
This is used just to append the backslash? Please just use `PathHelper.EnsureTrailingSlash`.